docs/run_in_existing_vpc.md

```diff
@@ -63,7 +63,7 @@ but it's better to avoid the later confusion!)
 
 ## Running in a shared subnet
 
-You can also use a shared subnet. Doing so is not recommended unless you are using external networking.
+You can also use a shared subnet. Doing so is not recommended unless you are using external networking (weave or flannel or calico or kope-routing etc).
```

Comment:
There are two different types.  You need CNI or External ... and reference the networking doc.